doc/api/errors.md

```diff
@@ -1516,6 +1516,13 @@ socket, which is only valid from a client.
 
 An attempt was made to renegotiate TLS on a socket instance with TLS disabled.
 
+<a id="ERR_TRACE_EVENTS_UNAVAILABLE"></a>
+### ERR_TRACE_EVENTS_UNAVAILABLE
+
+An attempt to load the `trace_events` module was made but the Node.js binary
+was compiled with the `NODE_USE_V8_PLATFORM=0` configuration option causing
+Trace Events to be unavailable.
```

Comment:
Optional suggestion:

Simplify this to something like:

> The `trace_events` module cannot be loaded because Node.js was compiled without Trace Events.

...or:

> The `trace_events` module could not be loaded because Node.js was compiled with V8 platform disabled.

This will allow for the possibility now or in the future that there are other reasons Node.js might be compiled without Trace Events.

(Totally non-blocking. Just a suggestion.)